src/clp_ffi_py/.clang-format

```diff
@@ -4,13 +4,17 @@ IndentExternBlock: "NoIndent"
 IncludeCategories:
   # NOTE: A header is grouped by first matching regex
   # Python header must be included before any others
-  - Regex: "^<clp_ffi_py/Python\\.hpp>$"
+  - Regex: "^<wrapped_dependency_headers/Python\\.hpp>$"
     Priority: -1
   # Project headers
   - Regex: "^<clp_ffi_py"
     Priority: 4
   # Third-party headers. Update when adding new third-party library
-  - Regex: "<(clp|gsl|json|msgpack|outcome)"
+  - Regex: "<(clp|gsl|json|outcome)"
+    Priority: 3
+  # Third-party headers with wrappers to silent clang-tidy warnings. Update when adding new
+  # third-party library
```

Comment:
```suggestion
  # Third-party facade headers wrapped with IWYU exports to resolve clang-tidy violations
```

Reply:
We don't need the second part since even if we added another library with a facade, we wouldn't need to update this line, right?